chartjs-chart-sankey.esm.d.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  CartesianScaleTypeRegistry,
+  Chart,
+  ChartComponent,
+  DatasetController,
+  SankeyParsedData, ScriptableContext,
```

Comment:
You can import the things you declare later? 👀

Reply:
Sure. Why not?